Milestone3/src/components/recipeInput.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+//not sure whether it was necessary to abstract the input fields as a component but I figured it couldn't hurt
+export default function RecipeInput({
```

Comment:
It might be a bit overkill! But an additional component is great practice and the code itself for it looks really good!